src/XMakeBuildEngine/Definition/Project.cs

```diff
@@ -1191,6 +1186,11 @@ public List<ProvenanceResult> GetItemProvenance(string itemToMatch, string itemT
         /// </param>
         public List<ProvenanceResult> GetItemProvenance(ProjectItem item)
         {
+            if (item == null)
+            {
+                return Enumerable.Empty<ProvenanceResult>().ToList();
```

Comment:
Is there a reason to prefer `Enumerable.Empty<>().ToList()` over `new List<>()`?